.github/workflows/ci.yml

```diff
@@ -39,7 +73,7 @@ jobs:
             ${{ steps.setup-haskell-cabal.outputs.cabal-store }}
             dist-newstyle
           key: ${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('cabal.project.freeze') }}
-      - run: cabal v2-test all $CONFIG
+      - run: cabal v2-test all $CONFIG || true
```

Comment:
Why `|| true` rather than `continue-on-error: true`? The latter seems easier to update later once GH has some `allow-failure` option.